Library/Formula/libtelnet.rb

```diff
@@ -0,0 +1,39 @@
+require "formula"
+
+class Libtelnet < Formula
+  homepage "https://github.com/seanmiddleditch/libtelnet"
+  url "http://github.com/seanmiddleditch/libtelnet/tarball/0.21"
+  sha256 "add20e5dcb55c26148c99201f986238f4a877202a023ba60ccb14b4d1bc1e9be"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "zlib" => :recommended
+
+  stable do
+    patch :DATA
```

Comment:
Just break the ` url ` and ` sha256 ` into a new ` stable do ` block and then insert the ` patch :DATA ` line there.